src/main/java/racing/CarRacing.java

```diff
@@ -0,0 +1,64 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRacing {
+
+    private static final List<Car> cars = new ArrayList<>();
+
+    private static Display display;
+    private static Util util;
+    public CarRacing( Display display, Util util){
+        this.display = display;
+        this.util = util;
+    }
+
+    public void startRacing(){
+        displayInputNumberOfCar();
+        createCar( inputNumberOfCar());
+        displayNumberOfTry();
+        racingCount( inputNumberOfTry());
+    }
+
+    private void displayNumberOfTry() {
+        display.printLine("시도할 회수는 몇 회 인가요?");
+    }
+
+    private void displayInputNumberOfCar(){
+        display.printLine("자동차 대수는 몇 대 인가요?");
+    }
+    private int inputNumberOfCar(){
+        int numberOfCar = util.inputNumber();
+        return numberOfCar;
+    }
+    private int inputNumberOfTry(){
+        int racingCount = util.inputNumber();
+        return racingCount;
+    }
+
+    protected int createCar(int numberOfCar){
+        for( int i=0;i<numberOfCar;i++){
+            cars.add(new Car());
+        }
+        return cars.size();
+    }
+    private void racingCount(int racingCount) {
+        for(int i = 0; i < racingCount; i ++){
+            racingStart();
+            endTryRacing();
+        }
+    }
+
+    private void racingStart(){
+        for( int i = 0; i < cars.size() ; i ++){
```

Comment:
조건 혹은 반복문 사용시 다음과 같이 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
또한 단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다!

[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses

```suggestion
        for (int i = 0; i < numberOfCar; i++) {
            cars.add(new Car());
        }
        return cars.size();
    }
    private void racingCount(int racingCount) {
        for (int i = 0; i < racingCount; i++) {
            racingStart();
            endTryRacing();
        }
    }

    private void racingStart(){
        for (int i = 0; i < cars.size(); i++) {
```